src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -342,6 +345,8 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
               toggleSelection={toggleSelection}
               enableSelectAll={selectMultiple}
               latestVersionText={latestVersionText}
+              // Intentionally do not pass "setCurrentContainer" -- search does not use the tree so it has nothing to update
```

Comment:
Thanks for the comment, I'm certain to need the reminder when looking at this code in a few months